src/main/java/service/ReportService.java

```diff
@@ -0,0 +1,8 @@
+package service;
+
+import java.util.Map;
+import model.Fruit;
+
+public interface ReportService {
+    public String report(Map<Fruit, Integer> dataBase);
```

Comment:
```suggestion
    public String report();
```
storage is publicly accessible